lib/main.dart

```diff
@@ -390,7 +390,7 @@ class _RuneState extends State<Rune> {
             final theme = FluentTheme.of(context);
 
             Widget content = Container(
-              color: appTheme.windowEffect == WindowEffect.solid
+              color: (appTheme.windowEffect == WindowEffect.solid || Platform.isIOS)
```

Comment:
Windows related window effect logic is already defined in another place, please organize them together to make the code maintainable.

![image](https://github.com/user-attachments/assets/8f4df703-b176-445e-90a8-1b7be592a92e)